### ISSUE TITLE
Regarding the periodicity in the vesta file

### ISSUE THREAD

**Issue by MrMu666** [kumagai-group/oxygen_vacancies_db#1]
In "oxygen_vacancies_db_data" folder, the vesta files(such as "Ag2BiO3_Va_O1_0.vesta") do not appear to contain the correct periodic boundary data, and the atomic clusters are isolated from each other after conversion to cif files. May I ask why this is so?
![Snipaste_2024-03-25_19-43-57](https://github.com/kumagai-group/oxygen_vacancies_db/assets/51439084/7a0a2a56-b0a3-42cb-a8a9-1c9c20a54574)

**Comment by yuuukuma** [comment]
To emphasize the local structure, only the neighboring atoms are depicted. If you want to draw all the atoms, you can use the CONTCAR-finish.

**Comment by MrMu666** [comment]
> To emphasize the local structure, only the neighboring atoms are depicted. If you want to draw all the atoms, you can use the CONTCAR-finish.
Thank you very much for your guidance! Also, does the number of the O-atom in the "cell_info.txt" file correspond to the number of the O-oxygen space in the "charge0.csv"?

**Comment by MrMu666** [comment]
> To emphasize the local structure, only the neighboring atoms are depicted. If you want to draw all the atoms, you can use the CONTCAR-finish.

Thank you very much for your guidance! Also, does the number of the O-atom in the "cell_info.txt" file correspond to the number of the O-oxygen space in the "charge0.csv"?

**Comment by yuuukuma** [comment]
Yes. the site number is consistent between cell_info.txt and charge?.csv files.
